pages/linux/macchanger.md

```diff
@@ -11,10 +11,22 @@
 
 `macchanger --random {{interface}}`
 
+- Set interface to a random MAC, and pretend to be a [b]urned-[i]n-[a]ddress:
+
+`macchanger --random --bia {{interface}}`
+
+- Set interface to a random vendor MAC of same/any kind:
```

Comment:
```suggestion
- Set an interface to a random vendor MAC address of same/any kind:
```

Also, please clarify: same kind with regards to what?

Reply:
I will remove this example. I have tested it, and it the tool does not comply with the documentation. So this is a buggy feature.